files/debian-initd

```diff
@@ -68,7 +69,7 @@ log_daemon_msg () {
 #
 report_bug()
 {
-	echo $*
+	echo "$*"
```

Comment:
is it actually helpful?

Reply:
$1, $2 = "pear" "apple"

"$@" becomes "pear" "apple"

"$*" becomes "pear apple"

`$*` is not protected

Reply:
agreed -- would expand globs. not that we pass any in our strings afaik but indeed doesn't hurt

Reply:
:) actually my editor (Midnight Commander) only highlights `1>&2` not `>&2` :)